src/components/Comments/Comment.js

```diff
@@ -179,6 +252,13 @@ const Comment = ({
             </ButtonHightlight>
           </TouchableOpacity>
         )}
+        {/* <ButtonHightlight
+          onLongPress={handleOnLongPress}
+          style={[styles.btnBlock(comment.user.id === yourselfId), styles.btn]}
+          onPress={() => refSheet.current.open()}>
+          <MemoSendDM width={20} height={18} />
+        </ButtonHightlight> */}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The commented-out code should be removed if it's not needed. If it's a work in progress or will be used later, consider adding a TODO comment explaining why it's there.
```diff
- {/* <ButtonHightlight
-   onLongPress={handleOnLongPress}
-   style={[styles.btnBlock(comment.user.id === yourselfId), styles.btn]}
-   onPress={() => refSheet.current.open()}>
-   <MemoSendDM width={20} height={18} />
- </ButtonHightlight> */}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->